### DISCUSSION TITLE
Scroll restoration not working with Infinite Queries in React Table.

### DISCUSSION THREAD

**Discussion by BalasubramaniM** [TanStack/query#2493] [General]
I'm currently using [InfiniteQueries](https://react-query.tanstack.com/guides/infinite-queries#_top) for onScroll pagination in my React Table V7 and everything works as expected. 

But when I get into some other page (in detail page from list view) and when I come back, my scroll position is lost in the Table and it comes back to the top. In the [Scroll Restoration](https://react-query.tanstack.com/guides/scroll-restoration) docs it is mentioned that 

> As long as your queries are being cached long enough (the default time is 5 minutes) and have not been garbage collected, scroll restoration will work out of the box all the time.

My cache time is 5 minutes in this case. Can anyone help me here?

https://react-query.tanstack.com/guides/infinite-queries#_top

**Comment by TkDodo** [comment]
react-query doesn't do anything specific for scroll restoration. It's just because the data comes from the cache, it is instantly displayed on the screen, so the browser can restore properly.

If you have a codesandbox reproduction of your situation, I can take a closer look at it.

**Reply by cubecleveland** [reply]
Hello, im looking for an example of how to do the infinte queries with react table. ?im having issues injecting the the useInifinteQuery data into the react table ? anyone has a good example ?

@BalasubramaniM do i have to use pagination for this to work ?
